docs/mmctl_export_job_cancel.rst

```diff
@@ -0,0 +1,50 @@
+.. _mmctl_export_job_cancel:
+
+mmctl export job cancel
+---------------------
+
+Show export job
+
+Synopsis
+~~~~~~~~
+
+
+Show export job
```

Comment:
@gabor-boros should this be `Cancel export job`?

Reply:
Thanks @keithgg! Updated.